.codacy.yml

```diff
@@ -0,0 +1,7 @@
+---
+exclude_paths:
```

Comment:
I wish there was an `include_paths` option instead as we'll have to make sure to update this anytime another `examples_*` folder is added. Otherwise, looks great


https://docs.codacy.com/repositories-configure/codacy-configuration-file/